### PR TITLE
Added CH2OH + HCN training reactions to R_Add_Multiple_Bond

### PR DESCRIPTION
**Pull request by alongd**
Calculations done at CCSD(T)/aug-cc-pVTZ//wB97xd/def2TZVP

### CODE REVIEW

**Comment by alongd** [comment]
@mjohnson541, could you take a look?

**Comment by alongd** [comment]
@mjohnson541, could you take a look?

File: input/kinetics/families/R_Addition_MultipleBond/training/dictionary.txt [hunk 1]
```diff
@@ -83232,11 +83237,6 @@ multiplicity 2
 3 *3 H u0 p0 c0 {2,S}
 4    H u0 p0 c0 {2,S}
 
-CHN-2
-1 *1 N u0 p1 c0 {2,T}
-2 *2 C u0 p0 c0 {1,T} {3,S}
-3    H u0 p0 c0 {2,S}
```
Comment (mjohnson541) [inline input/kinetics/families/R_Addition_MultipleBond/training/dictionary.txt:83238]: As far as I can tell this didn't change can the git history reflect that?

**Comment by alongd** [comment]
@mjohnson541, I addressed the comment and rebased

**Pull request merged**